File: Programmers/Lv.0/이어 붙인 수.cc
// https://school.programmers.co.kr/learn/courses/30/lessons/181928

#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
  string hol = "", jjak = "";
  for (int i=0; i < num_list.size(); i++) {
    if (num_list[i] % 2 == 1)
      hol += to_string(num_list[i]);
    else
      jjak += to_string(num_list[i]);
  }
  return stoi(hol) + stoi(jjak);
}

File: Programmers/코딩기초트레이닝/JS/이어 붙인 수.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181928?language=javascript

function solution(num_list) {
    let res1 = Number(num_list.filter((num) => num % 2 === 1).join(""));
    let res2 = Number(num_list.filter((num) => num % 2 === 0).join(""));

    return res1 + res2;
}